src/helpers/user/user-helper.js

```diff
@@ -8,6 +9,145 @@ const principalStatusApiMap = {
   Inactive: 'disabled',
   All: 'all',
 };
+
+const getBaseUrl = (url) => {
+  if (isInt) {
+    return url.int;
+  } else if (isStage) {
+    return url.stage;
+  } else if (isITLessProd) {
+    return url.prod;
+  } else {
+    return '';
+  }
+};
+
+async function fetchBaseUrl() {
+  try {
+    const response = await fetch(`${insights.chrome.isBeta() ? '/beta' : ''}/apps/rbac/env.json`);
```

Comment:
This will be obsolete as we are moving away from preview/beta. It should be handled via feature flags. I'd like to learn about the FF capabilities in fedramp.

Reply:
As far as I can tell the URLs in this json were the same for preview and stable, so I've opted to remove the logic for now.